contribs/github-bot/config.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"github.com/gnolang/gno/contribs/github-bot/client"
+	c "github.com/gnolang/gno/contribs/github-bot/conditions"
+	r "github.com/gnolang/gno/contribs/github-bot/requirements"
+)
+
+// Automatic check that will be performed by the bot
+type automaticCheck struct {
+	Description string
+	If          c.Condition   // If the condition is met, the rule is displayed and the requirement is executed
+	Then        r.Requirement // If the requirement is satisfied, the check passes
+}
+
+// Manual check that will be performed by users
+type manualCheck struct {
+	Description string
+	If          c.Condition // If the condition is met, a checkbox will be displayed on bot comment
+	Teams       []string    // Members of these teams can check the checkbox to make the check pass
```

Comment:
For now, it’s only: the members of the specified teams, or if none are specified, the staff members with write access to the repo. I was thinking it would be cleaner to put users in dedicated teams if needed (e.g., `tm2-owners` or `gnovm-reviewers`, etc.). 

But we could easily add support for individual users if necessary.